src/parts/Sidebar/SidebarContent/index.tsx

```diff
@@ -84,11 +86,20 @@ const SidebarContent = React.forwardRef<Ref, Props>(({
             <TestnetBadge className='ml-2' />
           )}
         </div>
-        <Navigation
-          onSmallScreen={onSmallScreen}
-          className='mt-5' />
-        <SocialMediaContainer className='p-2' />
+        <div>
```

Comment:
This div is needed to keep the ad at the top of the sidebar with the navigation.